### PR TITLE
doc: Update manual document

### PR DESCRIPTION
**Pull request by karasusan**
Update manuals as a whole.

## TODO
- Update some figures
- Rename classes according to the manual

### CODE REVIEW

File: com.unity.renderstreaming/Documentation~/data-streaming.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+### Properties
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| **Local** | Channel is local or remote | `False` |
+| **Label** | Channel label | Empty |
+
+## `DataSender` component
+
+This component receives input events from Unity.
```
Comment (kannan-xiao4) [inline com.unity.renderstreaming/Documentation~/data-streaming.md:35]:
```suggestion
This component sends any data from Unity.
```

File: com.unity.renderstreaming/Documentation~/data-streaming.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Data Streaming
+
+Data streaming is using [`RTCDataChannel`](https://docs.unity3d.com/Packages/com.unity.webrtc@2.4/manual/datachannel.html) of the WebRTC package internally.
```
Comment (kannan-xiao4) [inline com.unity.renderstreaming/Documentation~/data-streaming.md:3]:
```suggestion
Data streaming is using [`RTCDataChannel`](https://docs.unity3d.com/Packages/com.unity.webrtc@latest/manual/datachannel.html) of the WebRTC package internally.
```

Comment (karasusan) [reply com.unity.renderstreaming/Documentation~/data-streaming.md:3]: Unfortunately, `@latest` is not worked correctly.

Work : https://docs.unity3d.com/Packages/com.unity.webrtc@latest
Not work : https://docs.unity3d.com/Packages/com.unity.webrtc@latest/manual/datachannel.html

File: com.unity.renderstreaming/Documentation~/audio-streaming.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Audio Streaming
+
+`AudioSender` class and `AudioReceiver` class are the base classes for sending and receiving audio. And various components are implemented using inheritance relationships with these class. Components are provided for the sender and receiver. There are multiple components for the sender side, but `AudioClipReceiver` can receive from them the sent video.
```
[the remark this answers is not in the document]
Comment (karasusan) [reply com.unity.renderstreaming/Documentation~/audio-streaming.md:3]: Surely, this sentence is not clear. I am removing it.

**Review by kannan-xiao4 — APPROVED** [review]

**Pull request merged**